scripts/perftune.py

```diff
@@ -779,10 +787,14 @@ def __learn_irqs(self):
         This is a slow method that is going to read from the system files. Never
         use it outside the initialization code.
         """
-        if self.nic_is_bond_iface:
-            return { slave : self.__learn_irqs_one(slave) for slave in filter(self.__dev_is_hw_iface, self.slaves) }
-        else:
-            return { self.nic : self.__learn_irqs_one(self.nic) }
+        nicIrqDic={}
```

Comment:
Also be consistent with suffixes: you already used `_dict` in similar circumstances above. 

s/nicIrqDic/nic_irq_dict/